src/storage/state.rs

```diff
@@ -131,26 +129,17 @@ impl Borrow<BTreeMap<String, String>> for WadmActorInfo {
 }
 
 /// A wasmCloud Actor
-// NOTE: We probably aren't going to use this _right now_ so we've kept it pretty minimal. But it is
-// possible that we could query wadm for more general data about the lattice in the future, so we do
-// want to store this
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
 pub struct Actor {
```

Comment:
Change this name if we can since it is core state stuff and the name shows up in various places